inst/qml/BainAncovaBayesian.qml

```diff
@@ -15,82 +15,118 @@
 // License along with this program.  If not, see
 // <http://www.gnu.org/licenses/>.
 //
-import QtQuick			2.8
-import QtQuick.Layouts	1.3
-import JASP.Controls	1.0
-import JASP				1.0
+
+import QtQuick							2.8
+import QtQuick.Layouts					1.3
+import JASP.Controls					1.0
+import JASP								1.0
 
 Form
 {
+
 	VariablesForm
 	{
-		AvailableVariablesList { name: "variablesList"}
-		AssignedVariablesList {
-			name: "dependent"
-			title: qsTr("Dependent Variable")
-			singleVariable: true
-			allowedColumns: ["scale"]
+
+		AvailableVariablesList 
+		{ 
+			name: 						"variablesList"
 		}
 
-		AssignedVariablesList {
-			name: "fixedFactors"
-			title: qsTr("Fixed Factors")
-			singleVariable: true
-			allowedColumns: ["ordinal", "nominal", "nominalText"]
+		AssignedVariablesList 
+		{
+			name: 						"dependent"
+			title: 						qsTr("Dependent Variable")
+			singleVariable: 			true
+			allowedColumns: 			["scale"]
 		}
 
-		AssignedVariablesList {
-			name: "covariates"
-			title: qsTr("Covariates")
-			allowedColumns: ["ordinal", "nominal", "scale"]
+		AssignedVariablesList 
+		{
+			name: 						"fixedFactors"
+			title: 						qsTr("Fixed Factors")
+			singleVariable: 			true
+			allowedColumns: 			["ordinal", "nominal", "nominalText"]
```

Comment:
Does this look ok in QtCreator?

Reply:
Yeah this looks okay in Qt
![image](https://user-images.githubusercontent.com/25059399/105992748-c5d6ef80-60a5-11eb-9683-02a41a09c32e.png)